index.html

```diff
@@ -289,6 +289,38 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact_form">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in <span>touch.</span>
+        </p>
+        <form action="#" class="form" method="get">
```

Comment:
Good job on the form :muscle: However you missed out one of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) you used  ```Get```instead of ```post``` method.